README.md

```diff
@@ -29,54 +61,95 @@ There are some configuration options for the sysadmins out there. All configurat
 * `path` *(string)* Path to the application data folder, which contains the private key, message attachment data (blobs) and the leveldb backend. Defaults to `$HOME/.ssb`.
 * `master` *(array)* Pubkeys of users who, if they connect to the Scuttlebot instance, are allowed to command the primary user with full rights. Useful for remotely operating a pub. Defaults to `[]`.
 * `logging.level` *(string)* How verbose should the logging be. Possible values are error, warning, notice, and info. Defaults to `notice`.
+* `remote` ... TODO (this was formerly undocumented but came up in conversation)
```

Comment:
this `remote` thing (naively) looks like redundency. If it's really needed should it not be derived from what's in  `config.connections`?

Reply:
That would make sense, are you thinking something like an index number on the `config.connections` array?

Reply:
Please don’t touch `Remote` just yet. It is used by nearly all clients. Most known for `remote = ws://...` by patchbay and liteclients.

I share the sentiment that in some future `connections` should handle this somehow but just removing it without solid tests will give a lot of people sleepless nights.